libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -389,10 +380,16 @@ void AP_Camera::setup_feedback_callback(void)
 failed:
 #endif // CONFIG_HAL_BOARD
 
-    if (!_timer_installed) {
-        // install a 1kHz timer to check feedback pin
-        hal.scheduler->register_timer_process(FUNCTOR_BIND_MEMBER(&AP_Camera::feedback_pin_timer, void));
+    int8_t dpin = hal.gpio->analogPinToDigitalPin(_feedback_pin);
+    if (dpin == -1)                {
```

Comment:
A bit too much whitespaces

Reply:
Ha! Not really sure how I did that, will fix.